### ISSUE TITLE
Metric Name suggest to show render

### ISSUE THREAD

**Issue by dmytrodubinin** [dangermccann/logseq-metrics#3]
Why I type Metric Name plugin suggest me visualize rendered block.
Look into screenshot:
![image](https://user-images.githubusercontent.com/8066458/202715821-ab3598a0-9734-468b-9e61-193c6c0040eb.png)

**Comment by dangermccann** [comment]
Good catch.  The renderers are normally not meant to be used on the page that stores the data.  I'll add in a check for this.  Thank you.
